.gitignore

```diff
@@ -60,3 +60,4 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+*.sqlite
```

Comment:
The caching syetem is creating the .sqlite file. Or should I just ignore it locally.